README.md

```diff
@@ -49,7 +43,14 @@ unless real values don't help users know what to change.
 -->
 
 ```hcl
-
+module "scc_wp" {
+  source  = "terraform-ibm-modules/scc-workload-protection/ibm"
+  version = "latest" # Replace "latest" with a release version to lock into a specific release
```

Comment:
instead of "latest" can you use "X.X.X" ? It seems "latest" was causing some confusion previously